ansible/roles/biocache-properties/tasks/main.yml

```diff
@@ -20,14 +20,12 @@
 
 - name: Download SDS layers
   get_url: url={{ sds_layers_url | default('http://biocache.ala.org.au/archives/layers/sds-layers.tgz') }} dest="{{data_dir}}/biocache/layers/sds-layers.tgz" force=yes
-  when: sds_enabled is defined and sds_enabled == true
```

Comment:
@sadeghim If you need this for appd, it may be simpler to add sds_enabled to the appd ansible file to to make all biocache ansible invocations download the sds-layers archive if they don't all need it